recipes/xege/all/test_package/example.cpp

```diff
@@ -0,0 +1,15 @@
+#include <ege.h>
+#include <windows.h>
+
+#ifdef _MSC_VER
+#pragma comment(lib,"msvcrtd.lib")
```

Comment:
WinDDK debug runtimes 😕 Why is this required?

Reply:
I don't know too. But if I don't do this, Conan CI will throw out a likage error comppains about `__imp_*`ref not found.

This problem doesn't appear during my local test.

Reply:
This code was in the very first commit... can you point me to the error you saw?

Reply:
Sorry, I've started a pr #5972 at an earlier time, and [here](https://github.com/conan-io/conan-center-index/pull/5972#issuecomment-864108888) I found this error. After that, I closed it and change a new example.

Reply:
Ah since you missed the cmake wrapper and it was only MT that fails... its possible this is no longer required (or maybe add it to the system libs so it works for consumers)

Reply:
I will check it now. Thanks for your advice.